docs/locales/zh_CN/LC_MESSAGES/tutorial/FedDataFrameDataLoader.po

```diff
@@ -40,160 +42,359 @@ msgid "Introduction to the interface"
+#~ "FedDataFrame, and we can now perform "
+#~ "some data processing on it."
+#~ msgstr "得到的\\ ``hdf``\\ 是一个FedDataframe，我们接下来可以对他做一些数据处理"
+
+#~ msgid ""
+#~ "- For horizontal scenarios, there is "
+#~ "``secretflow.vertical.read_csv`` `API "
+#~ "<https://www.secretflow.org.cn/docs/secretflow/en/source/secretflow.data.horizontal.html#secretflow.data.vertical.read_csv>`__"
+#~ " available."
+#~ msgstr ""
+#~ "1. 对于水平场景有\\ ``secretflow.horizontal.read_csv`` `API"
+#~ " "
+#~ "<https://www.secretflow.org.cn/docs/secretflow/en/source/secretflow.data.horizontal.html#secretflow.data.horizontal.read_csv>`__\\"
+#~ " 。"
+
+#~ msgid ""
+#~ "In SecretFlow, we provide an interface"
+#~ " similar to ``pandas.read_csv`` to read "
+#~ "CSV data from different parties and "
+#~ "unify it into a federated concept "
+#~ "of data."
+#~ msgstr ""
+#~ "我们在SecretFlow中提供了类似于\\ ``pandas.read_csv``\\ "
+#~ "的接口来将各方数据的CSV读取成为一个联邦概念的统一的数据。"
+
+#~ msgid ""
+#~ "- For horizontal scenarios, there is "
+#~ "``secretflow.horizontal.read_csv`` `API "
+#~ "<https://www.secretflow.org.cn/docs/secretflow/en/source/secretflow.data.horizontal.html#secretflow.data.horizontal.read_csv>`__"
+#~ " available. "
+#~ msgstr ""
+#~ "- 对于水平场景有\\ ``secretflow.horizontal.read_csv`` `API"
+#~ " "
+#~ "<https://www.secretflow.org.cn/docs/secretflow/en/source/secretflow.data.horizontal.html#secretflow.data.horizontal.read_csv>`__\\"
+#~ " 。"
+
+#~ msgid ""
+#~ "In SecretFlow, we provide an interface"
+#~ " similar to ``pandas.read_csv`` to read "
+#~ "CSV data from different parties and "
+#~ "unify it into a federated concept "
+#~ "of data.For horizontal scenarios, there "
+#~ "is ``secretflow.horizontal.read_csv`` `API "
+#~ "<https://www.secretflow.org.cn/docs/secretflow/en/source/secretflow.data.horizontal.html#secretflow.data.horizontal.read_csv>`"
+#~ " available.For horizontal scenarios, there "
+#~ "is ``secretflow.vertical.read_csv`` `API "
+#~ "<https://www.secretflow.org.cn/docs/secretflow/en/source/secretflow.data.horizontal.html#secretflow.data.vertical.read_csv>`"
+#~ " available."
+#~ msgstr ""
+
+#~ msgid ""
+#~ "In SecretFlow, we provide an interface"
+#~ " similar to ``pandas.read_csv`` to read "
+#~ "CSV data from different parties and "
+#~ "unify it into a federated concept "
+#~ "of data. - For horizontal scenarios, "
+#~ "there is ``secretflow.horizontal.read_csv`` `API "
+#~ "<https://www.secretflow.org.cn/docs/secretflow/en/source/secretflow.data.horizontal.html#secretflow.data.horizontal.read_csv>`__"
+#~ " available. - For horizontal scenarios, "
+#~ "there is ``secretflow.vertical.read_csv`` `API "
+#~ "<https://www.secretflow.org.cn/docs/secretflow/en/source/secretflow.data.horizontal.html#secretflow.data.vertical.read_csv>`__"
+#~ " available."
+#~ msgstr ""
+#~ "在SecretFlow中，我们提供了类似于`pandas.read_csv`的接口，可以从不同方读取csv数据，并统一成联邦概念的数据。\n"
+#~ "1. 对于水平场景有\n"
+#~ "2. 对于垂直场景有\n"
+
+#~ msgid "|d66fa44b1e2a4efab7448736c3655596|"
+#~ msgstr ""
+
```

Comment:
all these need to be removed.
ref to https://github.com/secretflow/secretflow/blob/main/CONTRIBUTING.md#documentation-update-and-its-multilingual-version